opentracing-spring-web-autoconfigure/src/main/java/io/opentracing/contrib/spring/web/autoconfig/WebClientTracingProperties.java

```diff
@@ -0,0 +1,45 @@
+package io.opentracing.contrib.spring.web.autoconfig;
+
+import io.opentracing.contrib.spring.web.client.RestTemplateSpanDecorator;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ * Configuration for tracing of HTTP clients.
+ * Supports RestTemplate and AsyncRestTemplate beans.
+ *
+ * @author Michal Dvorak
+ * @see RestTemplateTracingAutoConfiguration
+ * @since 4/5/18
+ */
+@ConfigurationProperties(WebClientTracingProperties.CONFIGURATION_PREFIX)
+public class WebClientTracingProperties {
+
+    public static final String CONFIGURATION_PREFIX = WebTracingProperties.CONFIGURATION_PREFIX + ".client";
+
+    /**
+     * When set to true (default), it enables automatic tracing of RestTemplate beans, as well as instances created using default RestTemplateBuilder bean.
+     * Does not affect instances created manually.
+     */
+    private boolean enabled = true;
+    /**
+     * Configures COMPONENT tag value, when default RestTemplateSpanDecorator.StandardTags bean is used.
+     * When custom RestTemplateSpanDecorator is provided, this configuration has no effect.
+     */
+    private String componentName = RestTemplateSpanDecorator.StandardTags.COMPONENT_NAME;
```

Comment:
Do you need this to be configurable?

Reply:
I don't have any objections to make it configurable, I am just wondering why do you want to override it. Do yo have any specific use case?

Reply:
I think after more digging into spec that you are correct, there is no real use case to modify this name.
Other libraries have it hard-coded as well. I'll take that as convention and drop it.